fuel/src/test/kotlin/com/github/kittinunf/fuel/core/DataPartTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.kittinunf.fuel.core
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import java.io.File
+
+class DataPartTest {
+    private val currentDir = File(System.getProperty("user.dir"), "src/test/assets")
+
+    @Test
+    fun escapeContentDispositionFileName() {
+        val shortFile = File(currentDir, "lorem_ipsum_short.tmp")
+        val file = File(currentDir, "lorem_ipsum_short.tmp")
+        val filename = "malicious.sh\";\r\ndummy=a.txt"
+        val expectEscapedFilename = "malicious.sh%22;%0D%0Adummy=a.txt"
+
+        val specialCharInlinePart = InlineDataPart("first", name = "first", contentType = "application/json", filename = filename)
+        val specialCharFilePart = FileDataPart(shortFile, name = "second", filename = filename)
+        val specialCharBlobPart = BlobDataPart(file.inputStream(), name = "third", contentLength = file.length(), filename = filename)
+
+        assertThat(
+                "ContentDisposition filename must escape Double-Quote and CRLF",
+                specialCharInlinePart.contentDisposition == "form-data; name=\"first\"; filename=\"$expectEscapedFilename\""
+        )
+        assertThat(
+                "ContentDisposition filename must escape Double-Quote and CRLF",
+                specialCharFilePart.contentDisposition == "form-data; name=\"second\"; filename=\"$expectEscapedFilename\""
+        )
+        assertThat(
+                "ContentDisposition filename must escape Double-Quote and CRLF",
+                specialCharBlobPart.contentDisposition == "form-data; name=\"third\"; filename=\"$expectEscapedFilename\""
+        )
+
+        val normalFileName = "abc.txt"
```

Comment:
Could you please split this into another test function? This is to make sure that we have a small test case function testing only 1 thing.